### DISCUSSION TITLE
will rake tasks started in a cuke run outside VCR sandbox?

### DISCUSSION THREAD

**Discussion by tansaku** [vcr/vcr#878] [Q&A] [answered]
I've got what seems like a problem with http network requests still going through in a project, however VCR doesn't complain and there are no changes on the file system with the vcr cassettes.

I have feeling this all started when we made one of the cucumber steps hit a rake task to do an update.

I'm wondering if a rake task started off in the context of a cuke wrapped in VCR would run in a separate thread and not have VCR wrapping it?  Does that sound possible? And is there any way to deal with that other than not using rake in the cuke?

More details in https://github.com/AgileVentures/projectscope/issues/46

**Comment by krainboltgreene** [comment, marked as answer]
Yeah, that's a different process, it's only in-memory.
